server/command.go

```diff
@@ -29,12 +31,30 @@ func getCommand(client *pluginapi.Client) *model.Command {
 		DisplayName:          "MS Teams Meetings",
 		Description:          "Integration with MS Teams Meetings.",
 		AutoComplete:         true,
-		AutoCompleteDesc:     "Available commands: start, disconnect",
+		AutoCompleteDesc:     "Available commands: help, start, disconnect",
 		AutoCompleteHint:     "[command]",
 		AutocompleteIconData: iconData,
+		AutocompleteData:     getAutocompleteData(),
 	}
 }
 
+func getAutocompleteData() *model.AutocompleteData {
+	command := model.NewAutocompleteData("mstmeetings", "[command]",
+		"Available commands: start, disconnect")
+
+	help := model.NewAutocompleteData("help", "", "Display usage information")
+	command.AddCommand(help)
```

Comment:
Does this put the `help` command at the top of the list? If so, can we move it to the bottom? The order should be:

- start
- disconnect
- help

Also, we should list `help` in the original list on line 43